modular_zubbers/code/modules/storyteller/_events/fleshmind_event.dm

```diff
@@ -18,12 +18,16 @@
 
 /datum/round_event/fleshmind/start()
 	var/list/areas = list() // List of all the final areas
-	var/list/possible_spawn_areas = typecacheof(typesof(/area/station/maintenance, /area/station/security/prison, /area/station/construction)) // Get us our areas
-
+	// Blacklisted areas that the wireweed can't spawn in.
+	var/list/blacklisted_areas = typecacheof(typesof(
```

Comment:
WE CAN SPAWN IN DORMS BABY LETS GOOOO

Reply:
Nope, I just forgot to push the changes from my local branch.